.env.dev

```diff
@@ -0,0 +1,3 @@
+SECRET=b8TnKudJvyqcc2utzmc6Czruz54VR3zU
```

Comment:
It's not a problem for me that this secrets are part of the commit because it's only for local development